inc/namespace.php

```diff
@@ -46,6 +48,10 @@ function bootstrap() {
 
 	$is_alb_healthcheck = isset( $_SERVER['HTTP_USER_AGENT'] ) && strpos( $_SERVER['HTTP_USER_AGENT'], 'ELB-HealthChecker' ) === 0;
 
+	require_once __DIR__ . '/fluent_bit/namespace.php';
+	require_once __DIR__ . '/fluent_bit/error_handler/namespace.php';
+	Fluent_Bit\Error_Handler\bootstrap();
+
```

Comment:
Do we want to leave this as is or have it enabled behind a feature flag?

Reply:
I think it's an order of priority thing - if Fluent it is available we should use it if not then fall back. While releases aren't tied to infrastructure versions I think we'll need to be backwards compatible.

Make this conditional based on whether `Altis\Cloud\Fluent_Bit\is_available()` is true I suppose. That check could be done inside the `Fluent_Bit\Error_Handler\bootstrap()` function I suppose. No harm loading the PHP files even if they're unused.